libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/NotificationServiceConstants.java

```diff
@@ -17,6 +17,8 @@
 package com.yahoo.athenz.common.server.notification;
 
 public final class NotificationServiceConstants {
+    public static final String NOTIFICATION_PROP_SERVICE_FACTORY_CLASS = "athenz.zms.notification_service_factory_class";
```

Comment:
Kept the same property value even though it is now in Common to avoid breaking existing clients.